### PR TITLE
FHIR-45311

### PR DESCRIPTION
**Pull request by horsetriboulders**

### CODE REVIEW

**Review by yanheras — CHANGES_REQUESTED** [review]

File: input/profiles/StructureDefinition-computable-measure-cqfm.json [hunk 1]
```diff
@@ -85,7 +85,7 @@
             "key" : "cmp-4",
             "severity" : "error",
             "human" : "Improvement notation must be specified for scoring types other than cohort",
-            "expression" : "scoring.exists() and scoring in ('proportion' | 'ratio' | 'continuous-variable') implies improvementNotation.exists()"
+            "expression" : "scoring.exists() and scoring in ('proportion' | 'ratio' | 'continuous-variable') implies improvementNotation.exists() and must be specified at the root or group level based on where the scoring type is specified."
```
[the remark this answers is not in the document]
Comment (horsetriboulders) [reply input/profiles/StructureDefinition-computable-measure-cqfm.json:88]: Should it be something like this: 
"scoring.exists() and scoring in ('proportion' | 'ratio' | 'continuous-variable') implies group.select(extension('http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-improvementNotation').exists())"

**Comment by p9g** [comment]
Suggest testing with https://dev.fhirpath-lab.com/FhirPath tool

File: input/profiles/StructureDefinition-computable-measure-cqfm.json [hunk 2]
```diff
@@ -85,7 +85,7 @@
             "key" : "cmp-4",
             "severity" : "error",
             "human" : "Improvement notation must be specified for scoring types other than cohort",
-            "expression" : "scoring.exists() and scoring in ('proportion' | 'ratio' | 'continuous-variable') implies improvementNotation.exists()"
+            "expression" : "scoring.exists() and scoring in ('proportion' | 'ratio' | 'continuous-variable') implies group.select(extension('http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-improvementNotation').exists())"
```
Comment (p9g) [inline input/profiles/StructureDefinition-computable-measure-cqfm.json:88]: (scoring.exists() and scoring.coding.code in ('proportion' | 'ratio' | 'continuous-variable') and improvementNotation.exists()) xor
(group.where(extension('http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-scoring').where(exists() and value.coding.code in ('proportion' | 'ratio' | 'continuous-variable')).exists() and extension('http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-improvementNotation').exists()).exists())

**Comment by horsetriboulders** [comment]
Changes are made per request. Please review.

**Review by yanheras — APPROVED** [review]

**Pull request merged**